### ISSUE TITLE
ExtractorApi returns "undefined" for Item 9B

### ISSUE THREAD

**Issue by sash236** [janlukasschroeder/sec-api#26]
I ran the test app below for a URL and it returned "Undefined" for Item 9B. Item 9B had an entry on the 10-K
https://sec-api.io/docs/sec-filings-item-extraction-api-python-example

filing_url = "https://www.sec.gov/ix?doc=/Archives/edgar/data/0000004281/000000428120000038/form10k4q19.htm"

To ensure it is not licensing issue (c.f with #25), we experimented with an expired license and a good one. For the expired license it threw
`166             # request failed
--> 167             raise Exception("API error")`

But for the valid license it threw "undefined" which is incorrect.

TLDR, the API throws an undefined error for Item 9B.

**Comment by janlukasschroeder** [comment]
You want  to remove the trailing zeros of the CIK in the `filing_url`. Instead of using `0000004281`, try `4281`.

Example:

filing_url = "https://www.sec.gov/ix?doc=/Archives/edgar/data/4281/000000428120000038/form10k4q19.htm"
